doc/user/content/transform-data/patterns/durable-subscriptions.md

```diff
@@ -1,42 +1,45 @@
+title: "Durable subscription (Retain History)"
+description: "How to configure history retention to set up durable subscriptions in Materialize"
 menu:
   main:
     parent: 'sql-patterns'
-    name: "Time travel queries"
+aliases:
+  - /transform-data/patterns/time-travel-queries/
+  - /transform-data/patterns/time-travel-queries/#history-retention-period
 ---
 
-{{< private-preview />}}
+You can configure an object's [history retention
+period](#history-retention-period) for durable subscriptions.
 
-By default, all user-defined sources, tables, materialized views, and indexes
-keep track of the most recent version of their underlying data. Some use cases
-require accessing past versions of this data at specific points in time
-(i.e. time traveling). To allow time travel queries, you can configure these
-objects to **retain history**.
+{{< private-preview />}}
 
-When set to a value different than the default (i.e. no history retention), the
-history retention period of an object determines the duration for which
-historical versions of its underlying data are available for querying.
+## Durable subscriptions
 
-The common use cases for time traveling in Materialize are:
+Because [`SUBSCRIBE`](/sql/subscribe/) requests happen over the network, these
+connections might get disrupted for both expected and unexpected reasons. To
+avoid the need for re-snapshotting the data, you can:
 
-* Lossless, **durable subscriptions** to your changing results. See the
-  [durable subscriptions section below](#durable-subscriptions)
-  for details on how to create this type of subscription.
+- Adjust the [history retention period](#history-retention-period) for the
+  objects that a subscription depends on, and then
 
-* Accessing a past version of results at a specific point in time.
+- Use [`AS OF`](/sql/subscribe#as-of) to pick up where you left off on
+  connection drops.
 
 ## History retention period
 
-You can change the history retention period for [sources](/sql/create-source/),
-[tables](/sql/create-table/), [materialized views](/sql/create-materialized-view/),
-and [indexes](/sql/create-index/). To configure the history retention period for
-an object, use the `RETAIN HISTORY` option in the `CREATE` statement. This
-value can also be adjusted at any time using the object-specific `ALTER`
-statement.
+{{< private-preview />}}
 
-### Configuration
+The history retention period of an object ([sources](/sql/create-source/),
+[tables](/sql/create-table/), [materialized
+views](/sql/create-materialized-view/), and [indexes](/sql/create-index/))
+determines the duration for which historical versions of the object's underlying
+data are available for querying. By default, history retention period is unset
+for all user-defined sources, tables, materialized views, and indexes; that is,
+these objects keep track only of the most recent version of their underlying
+data. You can modify this retention period using the `RETAIN HISTORY` option.
```

Comment:
```suggestion
By default, all user-defined sources, tables, materialized views, and indexes
keep track of the most recent version of their underlying data. To gracefully
recover from connection disruptions and enable lossless, _durable
subscriptions_, you can configure the objects the subscription depends on
to **retain history**.

To configure the history retention period for an object, use the `RETAIN
HISTORY` option in its `CREATE` statement. This value can also be adjusted at
any time using the object-specific `ALTER` statement.
```

Reply:
Editing in GH, so unpolished, but the line of thought should be more focused on the functionality (durable subscriptions) rather than the implementation detail (history retention).